namespaces/live-1.cloud-platform.service.justice.gov.uk/cccd-dev/certificate.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: certmanager.k8s.io/v1alpha1
```

Comment:
Need to create a certificate once the zone is crated, so can you remove it in this PR

Reply:
👍 thats done